go.mod

```
@@ -4,6 +4,8 @@ module github.com/kubescape/storage
 
 go 1.21.3
 
+toolchain go1.21.5
+
```

question: why the differing toolchain, and in the patch versions, no less?

Why do we specifically need the `toolchain go1.21.5`? What’s the difference between the `1.21.3` and `1.21.5` that we explicitly require the latter? Is there something that prevents us from bumping the Go version to 1.21.5?

---

I hadn't explicitly updated this tbh.

I just ran `go mod tidy` and it put this in (which on a second thought wasn't there earlier).

I'm currently on version 1.21.5

---

Okay, so it should be safe to use:

```suggestion
go 1.21.5

toolchain go1.21.5

```

Please make the change and test it out.